### ISSUE TITLE
about nerf code

### ISSUE THREAD

**Issue by YinBo0927** [LabShuHangGU/FR-INR#3]
could you share the code on nerf-pytorch? thanks!

**Comment by YinBo0927** [comment]
I would like to know whether you use fr when use_viewdirs=True in nerf

**Comment by KexuanShi** [comment]
Hi, YinBo!

I think that I have adopted use_viewdirs=True in NeRF.  If you still have any questions about reproduction, feel free to consult me further.

**Comment by YinBo0927** [comment]
thanks for your good repo!

**Comment by YinBo0927** [comment]
![image](https://github.com/user-attachments/assets/f83618c8-909c-43ae-97a9-cdb3f1783e81)
I also want to know whether you modify the last three hiden layers like this or do you have other setting? thanks!

**Comment by KexuanShi** [comment]
That is correct. If you follow the previous learning rate schedule in Nerf-pytorch, alpha=0.005, 0.001, 0.01 will be better. If you still have any question about this repo, feel free to consult me further.

**Comment by YinBo0927** [comment]
thanks!!

**Comment by YinBo0927** [comment]
I have another question, why you just change the last three hidden layers of the nerf? thansk!

**Comment by KexuanShi** [comment]
Hi, YinBo!

Thanks for your continued interest in our work.

Reparameterizing the entire model would place higher demands on both runtime and hardware. Meanwhile, our experiments show that significant improvements can be achieved by reparameterizing just the last three hidden layers.

**Comment by YinBo0927** [comment]
thanks！！
